src/modules/schnorrsig/tests_exhaustive_impl.h

```diff
@@ -116,7 +116,8 @@ static void test_exhaustive_schnorrsig_verify(const secp256k1_context *ctx, cons
                         if (s <= EXHAUSTIVE_TEST_ORDER) {
                             secp256k1_scalar s_s;
                             secp256k1_scalar_set_int(&s_s, s);
-                            secp256k1_scalar_get_b32(sig64 + 32, &s_s);
+                            memset(sig64 + 32, 0, 32);
+                            sig64[63] = s;
```

Comment:
I think this is equivalent and shorter:

```suggestion
                            secp256k1_write_be32(sig64 + 32, s);
```

(also need to change `s_s` to `s` below)